### PR TITLE
CI: Save/Restore cache

### PR DESCRIPTION
**Pull request by Watson1978**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -59,8 +70,7 @@ shared: &shared
     - save_cache:
         paths:
           - ./vendor/bundle
-          - ./ccache
-        key: v1-dependencies-{{ checksum "Gemfile.lock" }}
```
Comment (Watson1978) [inline .circleci/config.yml]: `Gemfile.lock` was not included in this repository.
So `restore_cache` phase will be failed.

**Comment by Watson1978** [comment]
@mockdeep No response from Circle CI yet. Need to set anything?

**Comment by mockdeep** [comment]
@Watson1978 I'm sorry, I don't know what's happening here. One thing I did yesterday was to enable Github Checks for CircleCI, so maybe that has something to do with it. I was trying to avoid dozens of build jobs showing up under the checks section here. This turned out to be a messier transition than I was expecting. I'm going to spend the next few hours and try to get Github actions working.

**Comment by Watson1978** [comment]
hmm, `build_and_test` do not work on forked repository... ?

**Comment by mockdeep** [comment]
But build forked is enabled, and CircleCI has always worked in the past... Can you try [enabling Github Checks](https://circleci.com/docs/2.0/enable-checks/)?

build_forked

**Comment by Watson1978** [comment]
@mockdeep Thanks. I configured and hope it work well :)

**Pull request closed without merging**